tests/conftest.py

```diff
@@ -100,6 +100,43 @@ def users_fixture():
         })
     return deepcopy(users)
 
+denmark = {
+    'name': 'Denmark',
+    'invite_only': False,
+    'color': '#b0a5fd',
+    'pin_to_top': False,
+    'stream_id': 1000,
+    'in_home_view': True,
+}
+
+# This is a private stream;
+# only description/stream_id/invite_only/name/color vary from above
+secret_stream = {
+    'description': 'Some private stream',
+    'stream_id': 99,
+    'pin_to_top': False,
+    'invite_only': True,
+    'name': 'Secret stream',
+    'email_address': 'secret@example.com',
+    'color': '#c3c3c3',
+    'in_home_view': True,
+}
+
+
+@pytest.fixture(scope='module')
+def streams_fixture():
+    streams = [denmark, secret_stream]
```

Comment:
Is there a reason why you included denmark here?

Reply:
Yeah, I wanted to separate out the unique example for each fixtures. Like general stream, private stream etc. It will be helpful later if you just want to use one type of fixture in your test.